### PR TITLE
Add some more details

### PR DESCRIPTION
**Pull request by omnoid00**
Hello,
For the novice it is nice to have some additional information on the steps to follow. I can help with the documentation if ya'll agree.

### CODE REVIEW

File: installer-guide/mac-install.md [hunk 1]
```diff
@@ -4,38 +4,53 @@
...
+Once the task is completed exit the utility
+You will find the Install file in the directory
+Move the newly created image to Applications folder. It will simplify the next section.
 
 ![](/images/installer-guide/mac-install-md/gib-done.png)
 
 ## Setting up the installer
-
-Now we'll be formatting the USB to prep for both the macOS installer and OpenCore. We'll want to use macOS Extended(HFS+) with a GUID partition map. What this will do is create 2 partitions. The main `MyVolume` and a second called `EFI` which is used as a boot partition where your firmware will check for boot files.
+Note. Be aware that in some instances you may need a USB 2.0 stick and/or port.
```
Comment (khronokernel) [inline installer-guide/mac-install.md:51]: That's a port limit/missing ACPI definition issue, which are solved with XhciPortLimit and USBInjectAll on older firmwares

File: installer-guide/mac-install.md [hunk 2]
```diff
@@ -4,38 +4,53 @@
 * Supported version: 0.5.7
 
 While you don't need a fresh install of macOS to use OpenCore, some users prefer having a fresh slate with their boot manager upgrades.
+This guide works for a release of you choice as long as it is listed in the catalog
 
 **Note for legacy users**
 
-* If you want to use OpenCore on a system without UEFI, please follow the [Legacy Install](https://dortania.github.io/OpenCore-Desktop-Guide/extras/legacy.html) section first, after you can continue following the **Base folder structure** section
+* To use OpenCore on an older system with no UEFI (Plain Bios):
```
Comment (khronokernel) [inline installer-guide/mac-install.md:11]: > * To use OpenCore on an older system with no UEFI (Plain Bios):

Prefer this:
`* To use OpenCore on an older system with no UEFI support (Plain BIOS):`

File: installer-guide/mac-install.md [hunk 3]
```diff
@@ -4,38 +4,53 @@
...
+1. First follow the [Legacy Install](https://dortania.github.io/OpenCore-Desktop-Guide/extras/legacy.html) section
```
Comment (khronokernel) [inline installer-guide/mac-install.md:12]: Don't link to the site itself, please link to the file instead:
`[Legacy Install](/extras/legacy.md)`
This prevents the site from breaking if domains ever change

File: installer-guide/mac-install.md [hunk 4]
```diff
@@ -4,38 +4,53 @@
...
+2. Once completed, continue the guide at the **Base folder structure** section
 
-To start we'll want to grab ourselves a copy of macOS, you can skip this and head to formatting the USB if you're just making a bootable OpenCore stick and not an installer. For everyone else, you can either download macOS from the AppStore or with GibMacOS
+* If you are making an OpenCore bootable stick skip this section and head to **Formatting the USB**
 
 ## Downloading macOS
 
-Now lets grab [GibMacOS](https://github.com/corpnewt/gibMacOS) and run the `gibMacOS.command`:
+From a Mac OS machine that meets the requirements of the OS version you want to install, go directly to the Appstore and download the desired OS release the continue and continue to **Setting up the installer**
```
Comment (khronokernel) [inline installer-guide/mac-install.md:19]: Being picky but I'd like to use macOS, not Mac OS to fit with the branding better
> go directly to the Appstore and download the desired OS release the continue and continue to **Setting up the installer**

Think you made a typo, should look like this:
`go directly to the AppStore and download the desired OS, then continue to [**Setting up the installer**](/installer-guide/mac-install.md#setting-up-the-installer)`

File: installer-guide/mac-install.md [hunk 5]
```diff
@@ -4,38 +4,53 @@
...
+
+For machines that cannot download from the Appstore an image or for specific OS releases please use the GibMacOS utility.
```
Comment (khronokernel) [inline installer-guide/mac-install.md:21]: > For machines that cannot download from the Appstore an image or for specific OS releases please use the GibMacOS utility.

I'd prefer this:
`For machines that cannot download from the AppStore or are in need for a specific OS release, you can use the GibMacOS utility.`

File: installer-guide/mac-install.md [hunk 6]
```diff
@@ -4,38 +4,53 @@
...
+You will find the Install file in the directory
+Move the newly created image to Applications folder. It will simplify the next section.
 
 ![](/images/installer-guide/mac-install-md/gib-done.png)
 
 ## Setting up the installer
-
-Now we'll be formatting the USB to prep for both the macOS installer and OpenCore. We'll want to use macOS Extended(HFS+) with a GUID partition map. What this will do is create 2 partitions. The main `MyVolume` and a second called `EFI` which is used as a boot partition where your firmware will check for boot files.
+Note. Be aware that in some instances you may need a USB 2.0 stick and/or port.
+Make sure that there are no partitions in the stick.
```
Comment (khronokernel) [inline installer-guide/mac-install.md:52]: Seems a bit redundant as we're formatting the USB anyways, I don't think it's needed

File: installer-guide/mac-install.md (hunk 5 above)
Comment (1Revenger1) [reply installer-guide/mac-install.md:21]: `For machines that need a specific OS release or can't download from the AppStore, you can use the GibMacOS utility.` might be better @khronokernel 
This sentence is sort of awkward though.

File: installer-guide/mac-install.md [hunk 7]
```diff
@@ -4,38 +4,53 @@
...
+If you need beta versions of macOS, you can select `C. Change Catalog` which will give you additional options.
+In this example we will be installing Catalina and we will enter 1 at the prompt:
 
 ![](/images/installer-guide/mac-install-md/gib-process.png)
 
 This is going to take a while as we're downloading the entire 8GB+ macOS installer, so highly recommend reading the rest of the guide while you wait.
 
-Once finished, we'll next want to run the `BuildmacOSInstallApp.command`:
+Once the download is finished exit the utility but do no close the terminal window
+At the prompt run with "sudo" the utility `BuildmacOSInstallApp.command`: sudo BuildmacOSInstallApp.command
```
Comment (khronokernel) [inline installer-guide/mac-install.md:37]: > At the prompt run with "sudo" the utility `BuildmacOSInstallApp.command`: sudo BuildmacOSInstallApp.command

Don't forget to put 3 grave accents around the command (```)

File: installer-guide/mac-install.md (hunk 5 above)
Comment (khronokernel) [reply installer-guide/mac-install.md:21]: Ah yeah that's better

**Comment by khronokernel** [comment]
@omnoid00 Made my comments, once fixed I'll merge

File: installer-guide/mac-install.md (hunk 7 above)
Comment (1Revenger1) [reply installer-guide/mac-install.md:37]: Might be better as "At the prompt, run `sudo BuildmacOSInstallApp.command`" to run the BuildmacOSInstallApp.command". Or even just as "At the prompt, run `sudo BuildmacOSInstallApp.command`".

File: installer-guide/mac-install.md (hunk 1 above)
Comment (midi1996) [reply installer-guide/mac-install.md:51]: > That's a port limit/missing ACPI definition issue, which are solved with XhciPortLimit and USBInjectAll on older firmwares

Temporarily*

File: installer-guide/mac-install.md (hunk 7 above)
Comment (dhinakg) [reply installer-guide/mac-install.md:37]: Since when was `sudo` required? I don't recall needing it.

**Comment by omnoid00** [comment]
I guess it depends on your machine setup. I had to run it with sudo in the mac I used,otherwise it would not work.  But I could reword it as (you may need to use sudo)

________________________________

File: installer-guide/mac-install.md [hunk 8]
```diff
@@ -4,38 +4,53 @@
...
+* If you are making an OpenCore bootable stick skip this section and head to **Formatting the USB**
 
 ## Downloading macOS
 
-Now lets grab [GibMacOS](https://github.com/corpnewt/gibMacOS) and run the `gibMacOS.command`:
+From a Mac OS machine that meets the requirements of the OS version you want to install, go directly to the Appstore and download the desired OS release the continue and continue to **Setting up the installer**
+
+For machines that cannot download from the Appstore an image or for specific OS releases please use the GibMacOS utility.
+
+Now lets grab [GibMacOS](https://github.com/corpnewt/gibMacOS), unzip on a local directory.
```
Comment (muli) [inline installer-guide/mac-install.md:23]:
```suggestion
Now let's grab [GibMacOS](https://github.com/corpnewt/gibMacOS) and unzip to a local directory.
```

File: installer-guide/mac-install.md [hunk 9]
```diff
@@ -4,38 +4,53 @@
...
+Open a terminal window in the /gibMacOS/master directory and at the prompt run with "sudo" the `gibMacOS.command`: sudo gibMacOS.command
 
 ![](/images/installer-guide/mac-install-md/gib.png)
 
-From this, we get a nice list of macOS installers. If you need beta versions of macOS, you can select `C. Change Catalog`. For this example we'll choose 1:
+By default you get a list of the latest macOS installers.
```
Comment (muli) [inline installer-guide/mac-install.md:28]:
```suggestion
By default, you get a list of the latest macOS installers.
```

**Comment by khronokernel** [comment]
Took suggestions from this PR and added them here: https://github.com/dortania/OpenCore-Desktop-Guide/pull/80

**Pull request closed without merging**